apps/aecore/src/aec_chain_state.erl

```diff
@@ -452,6 +434,7 @@ assert_micro_signature(PrevNode, Node) ->
 -record(fork_info, { fork_id
                    , difficulty
                    , fees
+                   , key_hash
```

Comment:
it is not clear which key hash we have here - maybe call it latest_key_hash?

Reply:
sure, but that's same with fees 😉 anyways, I will change